src/azure-cli/azure/cli/command_modules/acs/_params.py

```diff
@@ -323,6 +323,9 @@ def load_arguments(self, _):
         c.argument('customer_admin_group_id', options_list=['--customer-admin-group-id'])
         c.argument('workspace_id')
 
+    with self.argument_context('openshift monitor enable') as c:
+        c.argument('workspace-id')
```

Comment:
add `help=The resource ID of an existing Log Analytics Workspace to use for storing monitoring data.`

Reply:
Done